### PR TITLE
CHE-490: add footer with product logo and support stuff

### PR DESCRIPTION
**Pull request by ashumilova**
**_2 Upvotes**_ Signed-off-by: Ann Shumilova ashumilova@codenvy.com

### CODE REVIEW

**Comment by ashumilova** [comment]
footer

File: dashboard/src/app/colors/che-color.constant.js [hunk 1]
```diff
@@ -22,6 +22,8 @@ export class CheColorsConfig {
   '$che-purple-color': '#4E5A96',
```
Comment (benoitf) [inline dashboard/src/app/colors/che-color.constant.js:22]: I think I've failed on the split as this file should be auto generated from the json file and then should be ignored

File: dashboard/src/assets/branding/product.json [hunk 2]
```diff
@@ -4,5 +4,6 @@
   "logoFile": "che-logo.svg",
   "favicon": "favicon.ico",
   "ideResources": "/ide/_app/",
-  "helpPath": "https://www.eclipse.org/che/"
+  "helpPath": "https://www.eclipse.org/che/",
+  "supportEmail": "support@codenvy.com"
```
Comment (benoitf) [inline dashboard/src/assets/branding/product.json]: should it be a codenvy mail there ?

**Comment by akurinnoy** [comment]
+1

**Comment by benoitf** [comment]
+1

**Pull request merged**